self-paced-labs/vertex-ai/vertex-ai-qwikstart/lab_exercise.ipynb

```diff
@@ -2,60 +2,72 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !tb-gcp-uploader --tensorboard_resource_name=$TB_INSTANCE_NAME \
</pre><p>This fails for me with the below error message. Probably we need to set up service account permissions, see <a href="https://cloud.google.com/vertex-ai/docs/experiments/tensorboard-training#create_a_service_account_with_required_permissions" target="_blank">here</a> </p><p> </p><pre class="ql-syntax" spellcheck="false">google.api_core.exceptions.PermissionDenied: 403 Permission 'aiplatform.tensorboards.get' denied on resource '//aiplatform.googleapis.com/projects/[...]/locations/us-central1/tensorboards/[...]' (or it may not exist).
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1570/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='106'/>

Reply:
Thanks, looking to replicate this on refresh instance. Sanjana and myself didn't encounter this. Maybe one of the APIs in the Readme.